src/SFA.DAS.AssessorService.Web/Controllers/AccountController.cs

```diff
@@ -179,22 +179,22 @@ public async Task<IActionResult> CreateAnAccount(CreateAccountViewModel vm)
             var inviteSuccess =
                 await _contactsApiClient.InviteUser(new CreateContactRequest(vm.GivenName, vm.FamilyName, vm.Email,null,vm.Email));
 
-            TempData["NewAccount"] = JsonConvert.SerializeObject(vm);
-
+            _sessionService.Set("NewAccount", JsonConvert.SerializeObject(vm));
```

Comment:
This looks like a PRG pattern, see the ModelStatePersistAttribute for a better way to do this

Reply:
No its not a PRG pattern since when a page gets refreshed there is no duplicate post. Just a simple get.